build_test_resources/linkfuntest.c

```diff
@@ -61,7 +62,7 @@ void my_app_simple_get_prop_greeting(ngx_link_func_ctx_t *ctx) {
             "200 OK",
             "text/plain",
             (char*) defaultGreeting,
-            strlen(defaultGreeting)
+            ngx_strlen(defaultGreeting)
```

Comment:
ngx_strlen is in nginx module not for apps, you could use any other library, It's compatible with C++ as well.